docker/checkout.bash

```diff
@@ -37,5 +37,5 @@ cd ${dir}/autoware.ai
 if [[ "$BRANCH" = "carma-develop" ]]; then
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
 else
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch carma-system-3.6.0
```

Comment:
Commit says Dockerfile and checkout.bash but I only see the checkout.bash change